drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/handlers/ScenarioSimulationGridPanelClickHandler.java

```diff
@@ -292,27 +296,52 @@ protected boolean manageLeftClick(final int canvasX,  final int canvasY, final b
         if (uiColumnIndex == null) {
             return false;
         } else {
-            return manageHeaderLeftClick(scenarioGrid, ap.getY(), uiColumnIndex);
+            return manageHeaderLeftClick(scenarioGrid, uiColumnIndex, ap);
         }
     }
 
     /**
      * This method check if the click happened on an <b>second level header</b> (i.e. the header of a specific column) cell. If it is so, manage it and returns <code>true</code>,
      * otherwise returns <code>false</code>
      * @param scenarioGrid
-     * @param gridY
      * @param uiColumnIndex
+     * @param rp
      * @return
      */
-    private boolean manageHeaderLeftClick(ScenarioGrid scenarioGrid, double gridY, Integer uiColumnIndex) {
-        ScenarioHeaderMetaData columnMetadata = getColumnScenarioHeaderMetaData(scenarioGrid, scenarioGrid.getModel().getColumns().get(uiColumnIndex), gridY);
+    private boolean manageHeaderLeftClick(ScenarioGrid scenarioGrid, Integer uiColumnIndex, Point2D rp) {
```

Comment:
I didn't found covered this method by any test, looking on its body seems to me as non trivial. Would you mind to cover it?

Reply:
@gitgabrio as we discussed on slack, think this can be tests https://github.com/gitgabrio/drools-wb/pull/2